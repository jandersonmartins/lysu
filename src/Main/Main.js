import * as S from './Main.styles'

export default S.Main
