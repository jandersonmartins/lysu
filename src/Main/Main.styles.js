import styled from 'styled-components'

const Main = styled.main`
  padding: 9rem 4rem 4rem 4rem;
`

export { Main }
