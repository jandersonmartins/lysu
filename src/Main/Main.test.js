import React from 'react'
import { render } from '@testing-library/react'
import Main from './Main'

test('renders correctly', () => {
  const { container } = render(
    <Main>
      <div>Somethig</div>
    </Main>
  )

  expect(container.firstChild).toMatchSnapshot()
})
