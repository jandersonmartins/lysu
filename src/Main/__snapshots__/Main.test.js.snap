// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  padding: 9rem 4rem 4rem 4rem;
}

<main
  class="c0"
>
  <div>
    Somethig
  </div>
</main>
`;
